apiserver/facade/interface.go

```diff
@@ -161,10 +160,10 @@ type ControllerDBGetter interface {
 	ControllerDB() (changestream.WatchableDB, error)
```

Comment:
We should try our hardest to get rid of this method. I can only see two reasons for it to exist.

 - Import
 - Export

In both cases, we could just create the import/export service much sooner.